clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/IngestionStatsReporter.java

```diff
@@ -142,8 +114,23 @@ protected void registerConditionalStats() {
         READY_TO_SERVE_WITH_RT_LAG_METRIC_NAME,
         new IngestionStatsGauge(this, () -> getStats().getReadyToServeWithRTLag(), 0));
 
+    if (!VeniceSystemStoreUtils.isSystemStore(storeName)) {
+      registerSensor(
+          NEARLINE_PRODUCER_TO_LOCAL_BROKER_LATENCY + "_rt_avg",
+          new IngestionStatsGauge(this, () -> getStats().getNearlineProducerToLocalBrokerLatencyAvg(), 0));
+      registerSensor(
+          NEARLINE_PRODUCER_TO_LOCAL_BROKER_LATENCY + "_rt_max",
+          new IngestionStatsGauge(this, () -> getStats().getNearlineProducerToLocalBrokerLatencyMax(), 0));
+      registerSensor(
+          NEARLINE_LOCAL_BROKER_TO_READY_TO_SERVE_LATENCY + "_rt_avg",
+          new IngestionStatsGauge(this, () -> getStats().getNearlineLocalBrokerToReadyToServeLatencyAvg(), 0));
+      registerSensor(
+          NEARLINE_LOCAL_BROKER_TO_READY_TO_SERVE_LATENCY + "_rt_max",
+          new IngestionStatsGauge(this, () -> getStats().getNearlineLocalBrokerToReadyToServeLatencyMax(), 0));
+    }
+
     // Do not need to check store name here as per user system store is not in active/active mode.
-    if (getStats().getIngestionTask().isActiveActiveReplicationEnabled()) {
+    if (null != getStats() && getStats().getIngestionTask().isActiveActiveReplicationEnabled()) {
```

Comment:
Why adding a null check here? Was it a bug before?

Reply:
While debugging I saw that in some cases getStats is null so we get a exception. This is just a safety check without changing any logic